launch_testing/launch_testing/test_runner.py

```diff
@@ -87,6 +87,7 @@ def run(self):
         self._test_run.bind(
             self._test_run.pre_shutdown_tests,
             injected_attributes={
+                'launch_service': self._launch_service,
```

Comment:
@hidmic Not directly related to this PR, but do you have any guidance for a process for deprecating 'injected_attributes' from launch_testing?  Something like 'put a warning in the next release when someone uses an injected attribute'  then in the release after that remove injected attributes?

I think injected_args are better and less confusing in just about every way.  They still require the user to know ahead of time what 'magic words' can be added as arguments to a test function, but beyond that they're way less mysterious

Reply:
Hmm, you could wrap them using a `property` that can generate a `warning` once if these are used.

Reply:
@hidmic Yeah - I know how to do the implementation side - my question is more about the administrative side.  Do we need a full release of "WARNING THIS WILL BE DEPRECATED" or is it sufficient to add the warning, them actually deprecate once all of OSRF's stuff is updated?